### ISSUE TITLE
Chore: Add Link to 'Frequently Asked Questions' in Landing Page Footer

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#343]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [x] Codebase
- [ ] User Interface
- [ ] Feature Request

## Description

We need to add a link to 'Frequently Asked Questions' in landing page footer.

Link: https://openpedia.netlify.app/faq/

**Comment by Bibek478** [comment]
Assign me this issue under jwoc24

**Comment by Sriparno08** [comment]
You're already assigned to another issue. Once your PR for it is merged, you will be considered for this issue. If anyone asks to be assigned before your PR is merged, he/she will be assigned.

**Comment by thatguywhocode** [comment]
Hey @Sriparno08 , assign me this task under JWOC.

**Comment by Sriparno08** [comment]
You're assigned.
